### PR TITLE
Combined User and Relayer into Wallet

### PR DESCRIPTION
**Pull request by danbryan**
## Description

This PR combined the User and Relayer into Commontype Wallet

### CODE REVIEW

File: interchain_test.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func TestInterchain_CreateUser(t *testing.T) {
 
 		user := ibctest.GetAndFundTestUserWithMnemonic(t, ctx, keyName, mnemonic, 10000, gaia0)
 		require.NoError(t, test.WaitForBlocks(ctx, 2, gaia0))
-		require.NotEmpty(t, user.Address)
+		require.NotEqual(t, "", user.Address)
```
Comment (DavidNix) [inline interchain_test.go:209]: nit:

A more suitable matcher here is `require.NotEmpty`

**Review by DavidNix — APPROVED** [review]
Nice cleanup!

File: interchain_test.go (hunk 1 above)
Comment (DavidNix) [reply interchain_test.go:209]: Or `require.NotZero` which in Go, refers to [zero values](https://go.dev/tour/basics/12).

Comment (danbryan) [reply interchain_test.go:209]: resolved with 
https://github.com/strangelove-ventures/ibctest/pull/255

**Comment by danbryan** [comment]
closes #220

**Pull request merged**